cmd/fyne_settings/settings/appearance.go

```diff
@@ -128,6 +129,8 @@ func (s *Settings) createPreview() image.Image {
 
 	c.SetContent(tabs)
 	c.Resize(fyne.NewSize(380, 380))
+	// wait for indicator animation
+	time.Sleep(150 * time.Millisecond)
```

Comment:
I think it would be better if this referenced the constant instead - `canvas.DurationShort`

Reply:
You are right, updated! :)